src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Auditable.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.metadata.persistence.entities;
+
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+import java.time.ZonedDateTime;
+
+@MappedSuperclass
+@EntityListeners(AuditingEntityListener.class)
+abstract class Auditable {
```

Comment:
Maybe instead of creating this new class, `BaseEntity` could be changed from interface to class? That way it wouldn't be necessary to make changes to every single entity class.

Reply:
OK! I have merged `BaseEntity` and `Auditable`.